<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cfgDialog</class>
 <widget class="QDialog" name="cfgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
         <item>
          <widget class="QGroupBox" name="baseOperGroup">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="readCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="progCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="blockProgCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="illegalBitCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="functionCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="verifyCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="eraseCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="blankCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="secureCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="otherOperGroup">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QCheckBox" name="insectionCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="pinCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="addressRelocateCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="IDCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="EEPROMCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="emptyBufferCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="protectCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="compareCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="masterCopyCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QCheckBox" name="vccVerifyCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="loopFunCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="onlineCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="unTestCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="enableSNCheck">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="checkSumGroup">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="sumCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="wordSumCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="crc16Check">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="crc32Check">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="bitsGroup">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="bit4Button">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="bit8Button">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="bit12Button">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="bit16Button">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="fileLoadGroup">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="wordAddressCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="bigEndianCheck">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>197</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>196</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
